java/README.md

```diff
@@ -14,3 +14,56 @@
 - Depth First Search
 - Breadth First Search
 - Binary Search
+
+### Data Structures:
+- Stack
+  - push(x): Push element x onto stack.  
+  - pop(): Removes the element on top of the stack.  
+  - top(): Get the top element.  
+  - empty(): Return whether the stack is empty.  
```

Comment:
`Push`, `Get`, and `Return` are inconsistent with comments for other operations as these are in the Imperative Mood (commands), while others are descriptive (`Removes`, `Inserts`, `Deletes`, `Returns`, etc).

Reply:
How would you call them?

Reply:
`Pushes`, `Gets`, `Returns` to be consistent with the other comments in the documentation.

However, for Python, the imperative mood (`Push`, `Get`, etc) is preferred.
```
The docstring is a phrase ending in a period. It prescribes the function or method's effect as a command ("Do this", "Return that"), not as a description; e.g. don't write "Returns the pathname ...".
```
https://www.python.org/dev/peps/pep-0257/#one-line-docstrings

Either is fine, but should be consistent.